### ISSUE TITLE
facebook/hubert-base-ls960

### ISSUE THREAD

**Issue by ajesujoba** [speechbrain/speechbrain#2667]
### Describe the bug

I trained several ASR models using different SSL models ([facebook/hubert-base-ls960](https://huggingface.co/facebook/hubert-base-ls960), [Orange/SSA-HuBERT-base-60k](https://huggingface.co/Orange/SSA-HuBERT-base-60k), [facebook/wav2vec2-xls-r-300m](https://huggingface.co/facebook/wav2vec2-xls-r-300m/tree/main)) that are publicly available, using the DVoice ASR CTC recipe. All the training was completed successfully, and the loss trend looks good. However, during inference using EncoderASR, one of the models, specifically Orange/SSA-HuBERT-base-60k, always returns an empty string.

### Expected behaviour

Like the other models that returned some text for the input audio, the ASR model trained with `Orange/SSA-HuBERT-base-60k` should do the same.

### To Reproduce

Run the DVoice recipe on `Orange/SSA-HuBERT-base-60k` on the Hausa split of the Swahili data, and perform inference on any audio input using 

`
asr_model = EncoderASR.from_hparams(source=model, savedir=tempf, run_opts={"device":"cuda"})
text = asr_model.transcribe_file(data)
print('text = ', text)
`

**Comment by TParcollet** [comment]
Hi, this is very hard for us to debug because 1. We need to finetune the model and 2. we don't have your recipe. The likelihood that we will reproduce your problem is very low. Could you, by any chance, provide us with 1. and 2. ?

**Comment by ajesujoba** [comment]
@TParcollet , thanks a lot for the quick reply. I have just added two models to a [google drive](https://drive.google.com/drive/folders/1FztgSKupywLnppc0LBgN42qXTy7MlJPB?usp=sharing). 
1. `hubert_bs_ls960`: this is an ASR model trained using [facebook/hubert-base-ls960](https://huggingface.co/facebook/hubert-base-ls960),
2. `ssa_hubert_60k`: this is the ASR model trained using [Orange/SSA-HuBERT-base-60k](https://huggingface.co/Orange/SSA-HuBERT-base-60k). This is the model that generates no response at all. 

Note that these ASR models are multilingual and they can be used for English, French, Arabic, portuguese, etc

Kindly let me know if you need any extra information and clarification

**Comment by ajesujoba** [comment]
@TParcollet, so sorry to bother you. Have you had chance to check the models?

**Comment by ajesujoba** [comment]
Thanks! Will appreciate if you have suggestions on what to try as we have a deadline sometime next week and need to evaluate the model

**Comment by Adel-Moumen** [comment]
> @TParcollet , thanks a lot for the quick reply. I have just added two models to a [google drive](https://drive.google.com/drive/folders/1FztgSKupywLnppc0LBgN42qXTy7MlJPB?usp=sharing).
> 
>     1. `hubert_bs_ls960`: this is an ASR model trained using [facebook/hubert-base-ls960](https://huggingface.co/facebook/hubert-base-ls960),
> 
>     2. `ssa_hubert_60k`: this is the ASR model trained using [Orange/SSA-HuBERT-base-60k](https://huggingface.co/Orange/SSA-HuBERT-base-60k). This is the model that generates no response at all.
> 
> 
> Note that these ASR models are multilingual and they can be used for English, French, Arabic, portuguese, etc
> 
> Kindly let me know if you need any extra information and clarification

Do you see this problem only on the Hausa split or is it also on other languages ? 

If you could @ajesujoba, can you please setup a small google colab so that I can run it? It would help me to speed up the process and find what is exactly the root of the problem. 

Thanks. :)

**Comment by ajesujoba** [comment]
Hi @Adel-Moumen, thanks 

I created a colab [here](https://colab.research.google.com/drive/14i29TezEz75hWFGmkN0qpR-4anbStxRv?usp=sharing) but it appears there are permission issues,.

**Comment by Adel-Moumen** [comment]
> Hi @Adel-Moumen, thanks
> 
> I created a colab [here](https://colab.research.google.com/drive/14i29TezEz75hWFGmkN0qpR-4anbStxRv?usp=sharing) but it appears there are permission issues,.

Hi, 

Thanks. Yes, could you please upload the audio file that you used on the google drive? I will download everything and run your model locally. 

Best.

**Comment by ajesujoba** [comment]
I have an audio file there already. I should add that. the models are multilingual ASR. But inference with SSA-Hubert returns empty response although its training log was good

**Comment by asumagic** [comment]
(Idle thought: could be an issue with incorrect sample rate conversion at inference?)

**Comment by ajesujoba** [comment]
I don't think it is about SR conversion. The training was done on audios with 16kHz SR, and for inference too we are using  the same SR. Both models I shared were trained on the same data, one does inference correctly, the other does not.

**Comment by TParcollet** [comment]
ping @Adel-Moumen on this one if he wants to have a look again. Sorry, to busy with the version 1.0.2 :/
